docs/extend/extend-apiml/api-mediation-routing.md

```diff
@@ -0,0 +1,102 @@
+# API Mediation Layer routing
```

Comment:
This is a new file so need to add it to the toc file `docs/.vuepress/pages.json`.